internal/app/common/user_id.go

```diff
@@ -0,0 +1,19 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/cloudoperators/heureka/internal/database"
+	"github.com/cloudoperators/heureka/internal/entity"
+)
+
+func GetUserId(db database.Database, uniqueUserId string) (int64, error) {
+	filter := &entity.UserFilter{UniqueUserID: []*string{&uniqueUserId}}
+	ids, err := db.GetAllUserIds(filter)
```

Comment:
"S0000000" is current placeholder, when OIDC auth will be ready we will have correct user name and these name will be existing entry in database.
Anyway we have an option to disable authentication so in this case it will be needed to have some "nobody", "anonymous", "guest" or "none" user, whatever we will name it.
Second problem will be authentication via JWT token, I guess we would need some scannerUser, or we would need to validate user name from JWT (or use systemuser for mentioned purpose).